micrometer-core/src/main/java/io/micrometer/core/instrument/binder/kafka/KafkaMetrics.java

```diff
@@ -212,36 +212,42 @@ else if (tags.size() == meterTagsWithCommonTags.size())
         }
     }
 
-    private Meter bindMeter(MeterRegistry registry, Metric metric, String name, Iterable<Tag> tags) {
-        Meter meter = registerMeter(registry, metric, name, tags);
+    private Meter bindMeter(MeterRegistry registry, Supplier<Map<MetricName, ? extends Metric>> metrics, MetricName metricName, String name, Iterable<Tag> tags) {
```

Comment:
I think improving the variable names a little here and in the other methods would improve readability:
- `metrics` to `metricsSupplier`
- `name` to `meterName`